### PR TITLE
Login

### PR DESCRIPTION
**Pull request by galta**
Just Login, 
Will add design with material-design with next task...

### CODE REVIEW

File: src/app/app.module.ts [hunk 1]
```diff
@@ -12,8 +18,14 @@ import { AppComponent } from './app.component';
   imports: [
     BrowserModule,
     FormsModule,
-    HttpModule
+    HttpModule,
+    RouterModule,
```
Comment (nirkaufman) [inline src/app/app.module.ts]: Double import of RouterModule

File: src/app/app.module.ts [hunk 2]
```diff
@@ -12,8 +18,14 @@ import { AppComponent } from './app.component';
...
+    RouterModule.forRoot(loginRoutes),
+    BrowserAnimationsModule,
+    MyCustomMaterialModule,
+    LoginModule
   ],
+  exports: [MyCustomMaterialModule],
   providers: [],
```
Comment (nirkaufman) [inline src/app/app.module.ts]: Remove empty providers array

File: src/app/login/login.module.ts [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+@NgModule({
+  imports: [
+    CommonModule,
+    RouterModule.forRoot(loginRoutes),
+    ReactiveFormsModule,
+    MyCustomMaterialModule
+  ],
+  providers: [LoginService],
+  declarations: [LoginComponent, LogoutComponent, PasswordRecoverComponent, RegistrationComponent],
+  exports: [LoginComponent, LogoutComponent, PasswordRecoverComponent, RegistrationComponent, MyCustomMaterialModule]
```
Comment (nirkaufman) [inline src/app/login/login.module.ts]: Why re-export the MyCustomMaterialModule?

File: src/app/login/login/login.component.ts [hunk 4]
```diff
@@ -0,0 +1,33 @@
...
+  ngOnInit() {
+    this.formLogin = new FormGroup({
+      email: new FormControl(null, [Validators.required, Validators.email]),
+      password: new FormControl(null, [Validators.required]),
+      rememberMe: new FormControl()
+    });
+  }
+
+  login() {
+    if (this.formLogin.valid) {
```
Comment (nirkaufman) [inline src/app/login/login/login.component.ts]: If not valid. what should happen?

File: src/index.html [hunk 5]
```diff
@@ -4,6 +4,9 @@
   <meta charset="utf-8">
   <title>MeetupControl</title>
   <base href="/">
+  <link href="../node_modules/@angular/material/prebuilt-themes/indigo-pink.css" rel="stylesheet">
```
Comment (nirkaufman) [inline src/index.html]: Do it inside the styles.css

File: src/app/login/login.module.ts (hunk 3 above)
Comment (galta) [inline src/app/login/login.module.ts]: Got it. Lazy loading issue....
It's Common/Shared Material components..

File: src/app/login/login/login.component.ts (hunk 4 above)
Comment (galta) [inline src/app/login/login/login.component.ts]: Need to show relevant message to user...

**Pull request merged**